### PR TITLE
RHDEVDOCS 6164 fix RN for 1.15.1 again, as chain.artifacts.pipelinerun.enable-deep-inspection allows both types

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to pipelines-docs-1.15, pipelines-docs-1.16

Issue:

RHDEVDOCS 6164

Link to docs preview:

https://81999--ocpdocs-pr.netlify.app/openshift-pipelines/latest/about/op-release-notes.html#new-features-1-15-1_op-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mramendi** [comment]
https://issues.redhat.com/browse/RHDEVDOCS-6164

**Review by ppitonak — APPROVED** [review]

**Review by enarha — COMMENTED** [review]
/lgtm

File: modules/op-release-notes-1-15.adoc [hunk 1]
```diff
@@ -460,7 +460,7 @@ With this update, {pipelines-title} General Availability (GA) 1.15.1 is availabl
 [id="new-features-1-15-1_{context}"]
 === New features
 
-* Before this update, in the `TektonConfig` CR, the `chain.artifacts.pipelinerun.enable-deep-inspection` spec supported only the `bool` value type. With this update, the `chain.artifacts.pipelinerun.enable-deep-inspection` spec supports only the `string` value type; the valid values for this spec are `"true"` and `"false"`.
+* Before this update, in the `TektonConfig` CR, the `chain.artifacts.pipelinerun.enable-deep-inspection` spec supported only the `bool` value type. With this update, the `chain.artifacts.pipelinerun.enable-deep-inspection` spec supports supports both the `bool` and `string` value types; if using the `string` value type, the valid values for this spec are `"true"` and `"false"`.
```
Comment (jkandasa) [inline modules/op-release-notes-1-15.adoc:463]: @mramendi typo: `supports supports` => `supports`

**Review by jkandasa — APPROVED** [review]
/lgtm

**Review by Srivaralakshmi — COMMENTED** [review]
/lgtm

**Pull request merged**